biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/WriteDependencies.scala

```diff
@@ -16,10 +16,14 @@ package nl.lumc.sasc.biopet.core
 
 import java.io.{File, PrintWriter}
 
+import akka.actor.ActorSystem
```

Comment:
Is the entire akka framework strictly necessary here? We are only doing HTTP calls right?